x-pack/plugins/maps/public/plugin.ts

```diff
@@ -36,11 +36,10 @@ import {
 import { featureCatalogueEntry } from './feature_catalogue_entry';
 // @ts-ignore
 import { getMapsVisTypeAlias } from './maps_vis_type_alias';
-import { registerLayerWizards } from './layers/load_layer_wizards';
 import { HomePublicPluginSetup } from '../../../../src/plugins/home/public';
 import { VisualizationsSetup } from '../../../../src/plugins/visualizations/public';
 import { MAP_SAVED_OBJECT_TYPE } from '../common/constants';
-import { MapEmbeddableFactory } from './embeddable';
+import { MapEmbeddableFactory } from './embeddable/map_embeddable_factory';
```

Comment:
don't pull from the `index.ts` file as it brings in the actual `Embeddable`. Just pull in the factory.